requirements.txt

```diff
@@ -5,12 +5,5 @@ psutil==5.9.0
 # From https://github.com/ultralytics/yolov5/blob/master/requirements.txt
 torch==1.11.0
 torchvision==0.12.0
-opencv-python==4.5.4.60
-pandas==1.1.4
-PyYAML==5.3.1
-Pillow==9.2.0
-tqdm==4.64.0
 onnx==1.12.0
```

Comment:
we don't need onnx, only onnxruntime

Reply:
however, we need numpy (yes it's included in torch / torchvision but it gets removed from this, we need to specify all direct dependencies)